### PR TITLE
Accessibility Escape

### PR DESCRIPTION
**Pull request by rustle**
Test Plan:
----------

Validated functionality using RNTester app and VoiceOver.

Release Notes:
--------------

[IOS] [ACCESSIBILITY] - Implements accessibility escape gesture prop

### CODE REVIEW

**Comment by rustle** [comment]
Documentation PR https://github.com/facebook/react-native-website/pull/655

File: RNTester/js/AccessibilityIOSExample.js [hunk 1]
```diff
@@ -26,6 +26,11 @@ class AccessibilityIOSExample extends React.Component<{}> {
         <View onMagicTap={() => alert('onMagicTap success')} accessible={true}>
           <Text>Accessibility magic tap example</Text>
         </View>
+        <View
+          onAccessibilityEscape={() => alert('onAccessibilityEscape success')}
```
[the remark this answers is not in the document]
Comment (rustle) [reply RNTester/js/AccessibilityIOSExample.js]: not sure what the expectation is here. this example code is consistent with the existing examples.

**Comment by shergin** [comment]
@rustle  Could you please rebase this on master?

**Comment by shergin** [comment]
... and make prettier happy?

**Comment by rustle** [comment]
> @rustle Could you please rebase this on master?

done

**Comment by rustle** [comment]
> ... and make prettier happy?

`npm run prettier` doesn't show any issues
`npm run lint` shows `22:37  warning  Unexpected alert  no-alert` but it's not clear what that means (to me)

**Comment by elucaswork** [comment]
@rustle thanks for adding it. Hopefully, it can be merged "soon". 🤞  
(I'm going to need it on my current project)

**Comment by cpojer** [comment]
Thanks for working on this. I'm gonna land this PR :)

**Pull request closed without merging**